plugins/PrivacyManager/templates/gdprOverview.twig

```diff
@@ -59,20 +59,20 @@
             Data retention for data stored in Matomo:
         </p>
         <ul>
-            {% if deleteLogsEnable %}
-                <li>all aggregated reports are deleted after <strong>{{ reportRetention }}</strong>.</li>
-            {% else %}
-                <li>aggregated reports are <strong>never</strong> deleted.</li>
-            {% endif %}
             {% if deleteReportsEnable %}
                 <li>all visits and actions raw data are deleted after <strong>{{ rawDataRetention }}</strong>.</li>
             {% else %}
                 <li>visits and actions raw data are <strong>never</strong> deleted.</li>
             {% endif %}
+            {% if deleteLogsEnable %}
```

Comment:
I just noticed, but are these variables named correctly? The `deleteReportsEnable` block talks about raw data & the `deleteLogsEnable` block talks about aggregated reports.

Reply:
seems I somehow screwed it up :see_no_evil: